src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -0,0 +1,33 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.HashMap;
+import java.util.Map;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "carts")
+public class Cart {
+    @Id
+    @NotNull
```

Comment:
Jeśli mamy zmienną typu "long" to i tak nie może być nullem, zresztą też jako ID nie będzie miałą wartości Null.
Proponował bym zmianę typu na Long i usunięcie adnotacji NotNull